server/cli/init/base-file-sources/hooks.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+/* eslint-disable no-var*/
+var cucumber = require('cucumber');
+var cucumberHtmlReport = require('cucumber-html-report');
+var log = require('npmlog');
+var moment = require('moment');
+var path = require('path');
+var Promise = require('bluebird');
+var fs = Promise.promisifyAll(require('fs'));
+
+
+function createReporter () {
+    /* eslint-disable new-cap */
+    var jsonFormatter = cucumber.Listener.JsonFormatter();
+    var outputDir = path.join(__dirname, '../', 'report')
+    jsonFormatter.log = jsonReportWriter;
+    return jsonFormatter;
+
+    function jsonReportWriter (content) {
+        var jsonFileName = getFileName('cucumberReport_', 'json');
+        var cucumberReport = path.join(outputDir, jsonFileName);
+        fs.writeFileAsync(cucumberReport, content)
+        .then(() => htmlReportWriter(cucumberReport))
```

Comment:
This is never transpiled, so it needs to be `function () { ... }` or it won't work in old Node.